scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1539,6 +1539,22 @@ public void sendRecord(Record record) {
 			public void sendFlight(DTLSFlight flight) {
 				sendHandshakeFlight(flight, connection);
 			}
+
+			@Override
+			public void pauseRetransmission() {
```

Comment:
`pause` and `resume` would make me beliefe, that the original retry timeout is continued. But in fact it's `cancel` an `restart`. Or did I oversee something?